### PR TITLE
iostat: add page

### PR DESCRIPTION
**Pull request by CJacob314**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- MacOS `iostat` version unknown

### CODE REVIEW

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks for you contribution! In our pages, we do not support general Markdown syntax (i.e. \*\*bold text \*\*). Could you please edit your PR as not to include such markup?

**Comment by CJacob314** [comment]
Sure, I'll revise my markdown according to that style as soon as I can.

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some suggestions for this page.

File: pages/osx/iostat.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Display only device statistics:
+
+`iostat -d`
+
+- Display incremental reports of CPU and disk statistics every 2 seconds:
+
+`iostat 2`
+
+- Display statistics for the first disk every second ad infinitum:
```
[the remark this answers is not in the document]
Comment (CJacob314) [reply pages/osx/iostat.md:18]: This [isn't](https://en.wikipedia.org/wiki/Ad_infinitum) a typo.

Comment (spageektti) [inline pages/osx/iostat.md:18]:
```suggestion
- Display statistics for the first disk every second indefinitely:
```
Not everyone may know what ad infinitum mean
```suggestion
- Display statistics for the first disk every second ad infinitum (indefinitely):
```

**Review by spageektti — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**